docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5459,6 +5474,11 @@ image](#create-an-image) section for more details.
                   - **Options** - key/value map of driver specific options
         - **StopGracePeriod** – Amount of time to wait for the container to terminate before
           forcefully killing it.
+        - **DNSConfig** – Specification for DNS related configurations in
```

Comment:
I don't see how godoc is relevant here. This is HTTP API documentation not go documentation.

The workflow will be "swagger spec" ->  "go code with comments" ->  "go doc".

This reference will be generated directly from the swagger spec. If we need godoc it will be available from the generated  comments.